pkgs/by-name/su/supercell-wx/package.nix

```diff
@@ -0,0 +1,154 @@
+    "AwsLevel*DataProvider.FindKeyFixed"
+    "AwsLevel*DataProvider.LoadObjectByKey"
+    "AwsLevel*DataProvider.Refresh"
+    "AwsLevel*DataProvider.GetAvailableProducts"
+    "AwsLevel*DataProvider.GetTimePointsByDate"
+    "AwsLevel*DataProvider.Prune"
+    "UpdateManagerTest.CheckForUpdates"
+    "WarningsProvider*\"https"
+
+    # These tests are failing (seemingly can't overwrite a file created by earlier test).
+    "SettingsManager/DefaultSettingsTest*"
+    "SettingsManager/BadSettingsTest*"
+  ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "supercell-wx";
+  version = "0.4.4";
+  src = fetchFromGitHub {
+    owner = "dpaulat";
+    repo = "supercell-wx";
+    rev = "refs/tags/v${finalAttrs.version}-release";
+    sha256 = "sha256-HghyRFLKboztQEOsvN2VaSxxcuPPu1GIakBNWRIWBOk=";
+    fetchSubmodules = true;
+  };
+
+  meta = {
+    homepage = "https://supercell-wx.rtfd.io";
+    downloadPage = "https://github.com/dpaulat/supercell-wx/releases";
+    description = "Live visualization of NEXRAD weather data and alerts";
+    longDescription = ''
+      Supercell Wx is a free, open source application to visualize live and
+      archive NEXRAD Level 2 and Level 3 data, and severe weather alerts.
+      It displays continuously updating weather data on top of a responsive
+      map, providing the capability to monitor weather events using
+      reflectivity, velocity, and other products.
+    '';
+    license = lib.licenses.mit;
+    mainProgram = "supercell-wx";
+    platforms = [ "x86_64-linux" ];
+    maintainers = with lib.maintainers; [ aware70 ];
+  };
+
+  CXXFLAGS = "-Wno-error=restrict -Wno-error=maybe-uninitialized -Wno-error=deprecated-declarations";
+  env.GTEST_FILTER = "-${lib.concatStringsSep ":" gtestSkip}";
+
+  doCheck = true;
+
+  # These tests aren't built by 'all', but ctest still tries to run them.
+  cmakeFlags = [
+    "-DCMAKE_CTEST_ARGUMENTS=-E;'test_mln_core|test_mln_widgets'"
+  ];
+
+  patches = [
+    # These are for Nix compatibility {{{
+    ./patches/use-find-package.patch # Replace some vendored dependencies with Nix provided versions
+    (substituteAll {
+      # Skip tagging build with git version, and substitute it with the src revision (still uses current year timestamp)
+      src = ./patches/skip-git-versioning.patch;
+      rev = finalAttrs.src.rev;
+    })
+    # Prevents using some Qt scripts that seemed to break the install step. Fixes missing link to some targets.
+    ./patches/fix-cmake-install.patch
+    # }}}
+
+    # These may be submitted upstream {{{
+    ./patches/add-cstdint.patch # use <cstdint> for GCC 13 compatibility
+    ./patches/fix-zoned-time.patch # fix ambiguity of some date/chrono functions in GCC 13
+    # fix apparent logic error when checking for supported audio codecs on launch
+    (fetchpatch {
+      name = "fix-audio-codec-setup.diff";
+      url = "https://patch-diff.githubusercontent.com/raw/dpaulat/supercell-wx/pull/223.diff";
```

Comment:
Usually we do not reference a PR directly. The content of the patch may change if the PR is updated. Since this PR is merged, you can fetch the patches from the master branch.